runtime/runtimetools/perfscripts/perf-hottest

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env python
+
+# ------------------------------------------------------------------------
+ # Copyright (c) 2020, 2020 IBM Corp. and others
+ #
+ # This program and the accompanying materials are made available under
+ # the terms of the Eclipse Public License 2.0 which accompanies this
+ # distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ # or the Apache License, Version 2.0 which accompanies this distribution and
+ # is available at https://www.apache.org/licenses/LICENSE-2.0.
+ #
+ # This Source Code may also be made available under the following
+ # Secondary Licenses when the conditions for such availability set
+ # forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ # General Public License, version 2 with the GNU Classpath
+ # Exception [1] and GNU General Public License, version 2 with the
+ # OpenJDK Assembly Exception [2].
+ #
+ # [1] https://www.gnu.org/software/classpath/license.html
+ # [2] http://openjdk.java.net/legal/assembly-exception.html
+ #
+ # SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+# ---------------------------------------------------------------------------
+
+# This script assumes the perf.data is in the folder where the script is;
+# This script also assumes perf-$pid.map files exist in /tmp
+# There is a known bug with the perf tool that make this script not usable. Details: https://bugzilla.kernel.org/show_bug.cgi?id=203811
```

Comment:
Is there any way to check the perf.data or perf --version or something to see if it is affected by this bug and report it to the user rather than leaving it as a comment in the script - this has a real potential to cause a lot of confusion in the field.

Reply:
With this bug, `perf record` will fail. Therefore the perf.data generate is not usable -- not only to this script, but other parsers as well. I'll update the comment to add more details